vm/src/org/mozartoz/truffle/nodes/local/InitializeTmpNode.java

```diff
@@ -8,11 +8,11 @@
 public class InitializeTmpNode extends OzNode implements FrameSlotNode {
 
 	@Child WriteFrameSlotNode writeFrameSlotNode;
-	@Child OzNode value;
+	@Child OzNode valueNode;
 
-	public InitializeTmpNode(FrameSlot slot, OzNode value) {
+	public InitializeTmpNode(FrameSlot slot, OzNode valueNode) {
 		this.writeFrameSlotNode = WriteFrameSlotNodeGen.create(slot);
-		this.value = value;
+		this.valueNode = valueNode;
```

Comment:
Ah, it's done below. But for the other use-case of InitializeTmpNode with a pattern match capture, it also seems good to stored deref-d values into the frame. So I think you could use `this.valueNode = DerefIfBoundNode.create(valueNode)` here.